src/domain/datasets/domain/src/messages/dataset_key_blocks_message.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use messaging_outbox::Message;
+use serde::{Deserialize, Serialize};
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+const DATASET_REFERENCE_OUTBOX_VERSION: u32 = 1;
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+/// Represents messages related to dataset key blocks.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub enum DatasetKeyBlocksMessage {
+    /// Message indicating that dataset key blocks have been introduced.
+    Introduced(DatasetKeyBlocksMessageIntroduced),
```

Comment:
This name makes an impression of "first introduction".
I guess we are rather "appending" than "introducing".

Reply:
Fair enough, I hadn't thought about it that way.

Renamed; Done